src/makeCalculator.js

```diff
@@ -37,7 +37,42 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
+    result: 0,
+
+    add(number) {
+      obj.result += number;
+
+      return this;
+    },
+    subtract(number) {
+      obj.result -= number;
+
+      return this;
+    },
```

Comment:
якщо вже ми не бачимо this
```suggestion
    subtract(number) {
      return obj.result - number;
    },
    
    or
    
    subtract(number) {
      obj.result -= number;
      
      return obj
    },
```